Source/AudibleUtilities/AudibleUtilities.csproj

```diff
@@ -5,10 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AudibleApi" Version="4.5.0.1" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <ProjectReference Include="..\..\..\audible api\AudibleApi\AudibleApi\AudibleApi.csproj" />
```

Comment:
revert. also update your references. I approved your audibleapi PR and upgraded the lucene nuget to upgrade that system drawing common v5

Reply:
Done

Reply:
And no you can `publish` all 4 versions successfully!